lib/view/view_templates/html_text_card_view_template.dart

```diff
@@ -13,17 +13,26 @@ class HtmlTextCardViewTemplate extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
-        backgroundColor: bgColor ?? AppColors.backgroundBlue,
-        appBar: AppBar(
-            backgroundColor: AppColors.appBarBackground,
-            iconTheme: Styles.appBarIconTheme,
-            title: Text(title, style: Styles.appBarTextStyle)),
-        body: SingleChildScrollView(
-            child: Padding(
-                padding: const EdgeInsets.all(12),
-                child: Card(
-                    child: HtmlWidget(html,
-                        textStyle: Styles.cardDescriptionTextStyle,
-                        bodyPadding: const EdgeInsets.all(6))))));
+      backgroundColor: bgColor ?? AppColors.backgroundBlue,
```

Comment:
maybe nicer to just set the default value on the bgColor param in constructor

Reply:
Will look at that, but yeap one of manu optimisations we still need to make due to lots and lots of duplication. But one step at a time.